configs/setups/awicm3/awicm3.yaml

```diff
@@ -91,22 +95,45 @@ general:
             - xios
             oasis_with_yac: True
             xios_interp_order: 1
+          v3.3:
+            major_version: v3.3
+            couplings:
+            - fesom-2.6+oifs-43r3-awicm-3.2+xios-2.5
+            add_include_models:
+            - xios
+            add_further_reading:
+            - oifs/oifs43.env.yaml
+            - xios/xios.env.yaml
+            - rnfmap/rnfmap.env.yaml
+            - fesom/fesom.env.yaml
```

Comment:
This is not needed anymore. One can remove it safely because they are now loaded form the component and they are kept. I will take care of removing them in a commit soon.